.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ jobs:
           command: "cd tdrs-backend; docker-compose down "
       - run:
           name: "Test: Run Unit Tests And Create Code Coverage Report"
-          command: "cd tdrs-backend; docker-compose run web sh -c \"pytest && pytest --cov-report=xml\""
+          command: "cd tdrs-backend; docker-compose run --rm web bash -c \"pytest --cov-report=xml\""
```

Comment:
These changes resolve an issue where test failures in the Circle CI build step would incorrectly return an exit code of `0` which would be treated by Circle CI as a passing result. There's a few things going on here:
* added the `--rm` flag - this enables docker-compose run to exit with the same code as the quoted command [ref](https://stackoverflow.com/a/37197988/9085243)
* changed the command shell from `sh` to `bash` - in most systems `sh` is a symlink to bash but when executed using `sh` it will run in POSIX mode, which introduces some changes to how exit codes are handled [ref](https://www.gnu.org/software/bash/manual/html_node/Bash-POSIX-Mode.html)
* removed duplicate executions of pytest - this was causing the tests to be run twice, and causing issues with how the exit codes are handled due to the way bash handles the `&&` (Logical AND)

I tested this resolves the issue of false positives by introducing an intentional communication error to the clamav tests and confirmed that the step was now failing as expected:
![Screenshot from 2021-04-28 15-56-55](https://user-images.githubusercontent.com/22626085/116467360-961ebb80-a83d-11eb-9168-843178b48a0c.png)